src/frontend/src/sol/types/network.ts

```diff
@@ -0,0 +1,11 @@
+import type { SolNetworkSchema } from '$sol/schema/network.schema';
+import { z } from 'zod';
+
+export type SolNetwork = z.infer<typeof SolNetworkSchema>;
+
+export enum SolanaNetworks {
+	MAIN_NET = 'mainnet',
```

Comment:
For consistency, please keep the network names as one word: `MAINNET`, `TESTNET`, etc.